### ISSUE TITLE
module 'tensorflow' has no attribute 'keras'

### ISSUE THREAD

**Issue by cainiaoiswo** [s0md3v/roop#219]
I got the following error when launching the program:
Traceback (most recent call last):
  File "D:\developer\roop\roop\run.py", line 15, in <module>
    from opennsfw2 import predict_video_frames, predict_image
  File "C:\Users\OMNIDREAM\AppData\Local\Programs\Python\Python310\lib\site-packages\opennsfw2\__init__.py", line 4, in <module>
    from ._inference import Aggregation
  File "C:\Users\OMNIDREAM\AppData\Local\Programs\Python\Python310\lib\site-packages\opennsfw2\_inference.py", line 15, in <module>
    from ._inspection import make_and_save_nsfw_grad_cam
  File "C:\Users\OMNIDREAM\AppData\Local\Programs\Python\Python310\lib\site-packages\opennsfw2\_inspection.py", line 17, in <module>
    model: tf.keras.Model,
AttributeError: module 'tensorflow' has no attribute 'keras'

**Comment by Luisdar0z** [comment]
yes... that also happened to me (I have cuda 11.6 btw), what I did was to uninstall tensorflow (pip uninstall tensorflow), delete it from the requirements.txt file and redo the installation process... 

1. pip install -r requirements.txt
2. pip uninstall onnxruntime onnxruntime-gpu
3. pip install torch torchvision torchaudio --force-reinstall --index-url https://download.pytorch.org/whl/cu116 
4. pip install onnxruntime-gpu

and with that it works for me so at the moment. 

(note that in step 3 I am downloading cu116 because it is the version of cuda I have installed, if you have cuda 11.8 you must install it as it is in the instructions)
